app/Svg/Database.hs

```diff
@@ -31,11 +31,15 @@ insertElements (paths, shapes, texts) = do
     mapM_ insert_ paths
     mapM_ insert_ texts
 
--- | Delete graphs from the database.
-deleteGraphs :: SqlPersistM ()
-deleteGraphs = do
-    runMigration migrateAll
-    deleteWhere ([] :: [Filter Text])
-    deleteWhere ([] :: [Filter Shape])
-    deleteWhere ([] :: [Filter Path])
-    deleteWhere ([] :: [Filter Graph])
+-- | Delete a graph with the given name from the database
+deleteGraph :: T.Text -> SqlPersistM ()
+deleteGraph graphName = do
+  runMigration migrateAll
```

Comment:
Oh we must have missed this in your last PR, but running the migration here is redundant and should be removed.